apps/api/src/app/environments-v1/usecases/output-renderers/email-output-renderer.usecase.ts

```diff
@@ -237,6 +239,29 @@ export class EmailOutputRendererUsecase {
     });
   }
 
+  /**
+   * Add the index to the liquid expression if it doesn't already have an array index
```

Comment:
This applies to the repeat block. If yes, please make the code comment more targeted for the future reader.